codeworld-api/src/CodeWorld/Color.hs

```diff
@@ -82,15 +79,15 @@ toHSL _ = Nothing
 mixed :: [Color] -> Color
 mixed colors = go 0 0 0 0 0 colors
   where go rr gg bb aa n ((fenceColor -> RGBA r g b a) : cs) =
-            go (rr + r^2 * a) (gg + g^2 * a) (bb + b^2 * a) (aa + a) (n + 1) cs
+            go (rr + r*r * a) (gg + g*g * a) (bb + b*b * a) (aa + a) (n + 1) cs
```

Comment:
Nit-picking: It's a little confusing to write `r*r * a`.  Just `r * r * a` makes the structure clearer.

Reply:
Fixed!